ecosystem/python/sdk/aptos_sdk/client.py

```diff
@@ -101,6 +101,19 @@ def info(self) -> Dict[str, str]:
     # Transactions
     #
 
+    def simulate_submit_bcs_transaction(self, signed_transaction: SignedTransaction) -> bool:
```

Comment:
probably better for this to just be simulate_bcs_transaction and take a raw transaction and an account?

Reply:
Cool !

Reply:
since I have your attention ;)

could we rename this to simulate_bcs_transaction?

🙏

Reply:
no problem

Reply:
After some exploration, I think it is better to just take in the signed tx and replace the signature with zero padding bytes ,which causes no safety issue.  
Morevoer,  it fits the process of python scripts users. Signed the tx, simulate the tx and if it pass, send the signed tx.
what do you think?

Reply:
Generally, it's better to not have people provide a real signature, unless they are expecting to actually use that signature.  Similar to if you signed a check, but then someone decided to copy the check and cross out your signature.  But, there's still your actual signature on a real check floating around.

It would be best to just provide an unsigned or raw transaction and take in a public key.